frontend/public/module/k8s/k8s-models.ts

```diff
@@ -72,3 +75,56 @@ export const modelFor = (ref: K8sResourceKindReference) => {
  * NOTE: This will not work for CRDs defined at runtime, use `connectToModels` instead.
  */
 export const allModels = () => k8sModels;
+
+/**
+ * Use this hook to find the model for resources using only group and kind
+ */
+export const useModelFinder = () => {
+  const models: K8sKind[] = _.values(
```

Comment:
I'm confused... Isn't this already an array? Why do we need `_.values`?

Reply:
This is a Map of a ModelReference to Model.